### PR TITLE
[14.0][MIG] hr_leave_custom_hour_interval: Migration to 14.0

### PR DESCRIPTION
**Pull request by etobella**
Ported from OCA/hr

This allows to select the time start by minutes, not only by half hours

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The name is misleading. It should be `hr_leave_custom_hour_interval`. Please take the occasion to rename it.

**Review by MiquelRForgeFlow — APPROVED** [review]

File: hr_leave_custom_hour_interval/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Hr Leave Hour",
```
Comment (MiquelRForgeFlow) [inline hr_leave_custom_hour_interval/__manifest__.py]:
```suggestion
    "name": "Hr Custom Hour Interval",
```

**Review by kluna1998 — APPROVED** [review]

**Review by sanchonuria — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Peril, it will be merged twice! 🫣

**Pull request merged**